dcicutils/qa_checkers.py

```diff
@@ -425,7 +432,4 @@ def summarize(problems):
             n += len(matches)
             detail += f"\n In {file}, {summarize(matches)}."
         message = f"{n_of(n, 'problem')} detected:" + detail
-        if if_used == 'warning':
-            warnings.warn(message, category=warning_category, stacklevel=2)
-        else:  # if_used == 'error'
-            raise AssertionError(message)
+        raise AssertionError(message)
```

Comment:
Was the `if_used` behavior here removed intentionally? If so docstring should be updated

Reply:
Yeah, it's intentional. It was recently added and no one is using it yet. I found a place higher in the call chain to do this effect more simply. I'll adjust doc string.